tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -1513,6 +1516,79 @@ def test_conv_core_nondivis(
     )
 
 
+# The following test takes various shape sizes from resnet50, unet and stable diffusion and tests for different number of groups - all the way to num_groups = num_in_channels (depthwise conv)
+@skip_for_grayskull()
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 16384}], indirect=True)
+@pytest.mark.parametrize(
+    "batch_size, input_channels, output_channels, input_height, input_width, filter_height, filter_width, stride_h, stride_w, pad_h, pad_w, groups, use_1d_systolic_array, config_override, use_shallow_conv_variant",
+    ((1, 64, 64, 16, 16, 3, 3, 1, 1, 2, 2, 2, True, None, False),),
+)
+@pytest.mark.parametrize(
+    "weights_dtype",
+    [ttnn.bfloat16],
+)
+@pytest.mark.parametrize(
+    "activations_dtype",
+    [ttnn.bfloat8_b, ttnn.bfloat16],
+)
+@pytest.mark.parametrize("math_fidelity", [ttnn.MathFidelity.LoFi])
+@pytest.mark.parametrize("output_layout", [ttnn.TILE_LAYOUT])
+@pytest.mark.parametrize(
+    "dilation",
+    [
+        2,
+    ],
+)
+@pytest.mark.skip("Dilation is WIP")
+def test_conv_dilation(
```

Comment:
why is this test part of this PR and not the dilation one?

Reply:
It had started with implementing dilation when I realized conv_params needed to be replaced. This test case is currently skipped.